example/index.html

```diff
@@ -8,7 +8,7 @@
   <script type="module" src="/index.js"></script>
 </head>
   <body>
-    <diamond-container>
+    <diamond-wrap size="tablet" gutter="md">
```

Comment:
`size="tablet"` is a bit strange, it's the size of a breakpoint and the breakpoints aren't exposed.

Should we have more standard sizes (sm, md, lg) instead then have the wrap sizes in a CSS variable so they can be optionally overridden?

Reply:
Yeah, that's probably more flexible